lib/plugins/pluginbase.js

```diff
@@ -91,6 +93,17 @@ function init (majorPills, minorPills, statusPills, bgStatus, tooltip) {
     }
   };
 
+  pluginBase.addForecastPoints = function addForecastPoints (points, sbx) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['sbx' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/772273510/issues/source?bid=2452379&fileBranchId=2479882#l96)